### PR TITLE
Issue #13663: EJB Persistent Timer feature for Jakarta

### PR DESCRIPTION
**Pull request by tkburroughs**
Add enterpriseBeansPersistentTimer-4.0 and basic FAT

for #13663

### CODE REVIEW

**Comment by tkburroughs** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansPersistentTimer-4.0/resources/l10n/io.openliberty.enterpriseBeansPersistentTimer-4.0.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the use of persistent timers in Enterprise JavaBeans.
```
Comment (brideck) [inline dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansPersistentTimer-4.0/resources/l10n/io.openliberty.enterpriseBeansPersistentTimer-4.0.properties:17]: Should we still be using Enterprise JavaBeans here in the description?

Comment (tkburroughs) [reply dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansPersistentTimer-4.0/resources/l10n/io.openliberty.enterpriseBeansPersistentTimer-4.0.properties]: Nice catch; thanks.  I've updated all the translations to use Jakarta Enterprise Beans and also fixed the copyright.

**Comment by tkburroughs** [comment]
Updated:
- corrected copyrights
- correctly used Jakarta Enterprise Beans
- added `WLP-Activation-Type: parallel` (all new features should use this)

**Comment by tkburroughs** [comment]
#run-libby-bot

**Comment by tkburroughs** [comment]
#build

**Review by brideck — APPROVED** [review]

**Pull request merged**